src/core/tests/type_prop/bitwise_and.cpp

```diff
@@ -9,3 +9,4 @@
 using Type = ::testing::Types<ov::op::v13::BitwiseAnd>;
 
 INSTANTIATE_TYPED_TEST_SUITE_P(type_prop_bitwise_and, BitwiseOperator, Type);
+INSTANTIATE_TYPED_TEST_SUITE_P(type_prop_bitwise_and, BitwiseOperatorBoolean, Type);
```

Comment:
seems out-of-scope change

Reply:
This is in scope, those ops have the same base class and can reuse the same test suite, only the test case with boolean type can't be reused, so I moved them to BitwiseOperatorBoolean tests and instantiated separately.